### DISCUSSION TITLE
use prosemirrorNodeToHtml translate the dom，always wrapped by the p tag

### DISCUSSION THREAD

**Discussion by Just-focus** [remirror/remirror#1234] [Q&A]
```js
const content = prosemirrorNodeToHtml(state.doc)
```

![image](https://user-images.githubusercontent.com/18374969/133190396-443b55d6-e66a-4350-80ec-63b666b739f9.png)

when i use prosemirrorNodeToHtml method to translate the dom, it will be wrapped by the p tag, how can i remove the p tag or use the custom tag ( like div ) to replace

**Comment by ocavue** [comment]
In most cases, a `doc` node contains multiple `paragraph` nodes. `paragraph` nodes will become `<p>` when using `prosemirrorNodeToHtml` by default. May I ask why do you want to remove or replace `<p>`?
